cli/src/commands/test/gas_report.rs

```diff
@@ -0,0 +1,173 @@
+// Copyright 2023 Polyphene.
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::collections::BTreeMap;
+
+use comfy_table::{
+    modifiers::UTF8_ROUND_CORNERS, presets::UTF8_FULL, Attribute, Cell, Color, Table,
+};
+use kythera_lib::{DeployedActor, ExecutionEvent, Method, Payload, TestResult, TestResultType};
+
+/// Gas report for the tested contracts.
+#[derive(Default, Debug)]
+pub struct GasReport {
+    reports: BTreeMap<DeployedActor, ActorInfo>,
+}
+
+#[derive(Debug, Default)]
+/// Actor method calls information
+/// TODO: calculate actor deployment gas consumption.
+pub struct ActorInfo {
+    methods: BTreeMap<Method, Vec<u64>>,
+}
+
+/// A Method and its gas cost.
+struct MethodCost {
+    gas_cost: u64,
+    num: u64,
+    from: u64,
```

Comment:
I think `from` is a bit misleading as a name. Maybe `at` or `at_id` would make it clearer ?

Reply:
yeah you are right, thanks Thomas, updated to `at` instead